static/js/components/tiles/ranking_tile.tsx

```diff
@@ -269,6 +243,84 @@ function fetchData(
     });
 }
 
+// Reduces RankingData to only the SV used for sorting, to be compatible for multi-column rendering in RankingUnit.
+function transformRankingDataForMultiColumn(
+  rankingData: RankingData,
+  statVarSpecs: StatVarSpec[]
+): RankingData {
+  const svs = statVarSpecs.map((spec) => spec.statVar);
+  const sortSv = svs[svs.length - 1];
+  const sortedPlacePoints = rankingData[sortSv].points;
+  const svsToDict = svs.map((sv) => {
+    const placeToVal = {};
+    const points = rankingData[sv].points;
+    for (const p of points) {
+      placeToVal[p.placeDcid] = p.value;
+    }
+    return placeToVal;
+  });
+  for (const p of sortedPlacePoints) {
+    p.value = undefined;
+    p.values = svs.map((_, i) => svsToDict[i][p.placeDcid]);
```

Comment:
could just do `svsToDict.map(svDict => svDict[p.placeDcid])`?

Reply:
we still need the index (since svsToDict is a list also, not a dict).

Reply:
not sure why the index is needed? if i understand correctly, svsToDict is just a list of { [placeDcid]: value } where svsToDict[i] corresponds to svs[i] .. so we can just map each item in svsToDict to the value in that dict for p.placeDcid?